conductr_cli/sandbox_run.py

```diff
@@ -81,3 +84,9 @@ def wait_for_conductr(args, run_result, current_retry, max_retries, interval):
     # Reprint previous message with flush to go to next line
     log.progress(last_message, flush=True)
     return True if current_retry < max_retries else False
+
+
+def write_run_command():
+    os.makedirs(DEFAULT_CLI_TMP_DIR, mode=0o700, exist_ok=True)
+    with open(LATEST_SANDBOX_RUN_ARGS_FILE, mode='w') as f:
+        f.write(' '.join(sys.argv[2:]))
```

Comment:
No, it will not break. Every word, no matter if an arg key, a value or partially a value, is a separate element in the array. The `parser.parse_args`  function in `sandbox_main` does then do the parsing based on this array.

We just need to ensure that file contains the same arg string as when manually performing a `sandbox run`.